src/components/dashboard/constants.js

```diff
@@ -44,3 +44,5 @@ export const SM_NUM_COLUMNS = 2;
 export const SM_PIXELS = 600;
 export const XS_NUM_COLUMNS = 1;
 export const XS_PIXELS = 0;
+
+export const USER_SURVEY = "https://cyver.se/srvy";
```

Comment:
Is there a particular reason that we're using the short-url here instead of the fully-expanded link? (I guess there might be some sort of analytics being done via the short-url provider?). If there isn't one, I'd rather we had the full link, most likely, but I don't feel strongly about it if you'd prefer not.

Reply:
The survey is in google forms and has a pretty long url. So we shortened it.

Reply:
I guess I was figuring the length of the URL is pretty irrelevant if you're just clicking a button to go to it, especially as we have it here where it won't even show in the browser on hover (pretty sure). It's a short-term thing so there's not much risk of the shortlink disappearing, but it's still an extra DNS lookup and redirect for no practical reason I can think of.

But it's still a short-term/temporary thing so I'm not that bothered by it. It just seems a little untidy, haha.

Reply:
True that links won't show up on hover. Now that I think about it, we will also be sending this link out to users in email and intercom messages.

Reply:
Yeah -- I don't know if we're using the rebrandly analytics (appears to be what the cyver.se domain is pointing to). If we are, definitely keep the short link, in any case, and it's worth sending it out in email/intercom where the link will actually appear for sure.